x-pack/plugins/endpoint/public/embeddables/resolver/factory.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { ResolverEmbeddable } from './';
+import {
+  EmbeddableFactory,
+  EmbeddableInput,
+  IContainer,
+} from '../../../../../../src/plugins/embeddable/public';
```

Comment:
is there a standard for importing stuff from `kibana` v. doing a relative path import?

Reply:
@stacey-gammon might know the answer

Reply:
Right now the recommendation is relative paths.  There is some ongoing work/discussion about supporting a better alternative, but it isn't finished yet.  Follow https://github.com/elastic/kibana/issues/40446 for progress.  Eventually, we hope that the recommendation can be to use `src/plugins/...`, which can work, but using that now can also kill your jest tests.

Reply:
i'm importing using tsserver exclusively so its no bother to me. has good compatibility w/ IDEs i think